src/qibocal/protocols/characterization/two_qubit_interaction/chevron.py

```diff
@@ -0,0 +1,251 @@
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional, Tuple, Union
+
+import numpy as np
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+from qibolab import AcquisitionType, AveragingMode, ExecutionParameters
+from qibolab.platform import Platform
+from qibolab.pulses import FluxPulse, PulseSequence, Rectangular
+from qibolab.sweeper import Parameter, Sweeper
+
+from qibocal.auto.operation import Parameters, Qubits, Results, Routine
+from qibocal.data import DataUnits
+
+
+@dataclass
+class ChevronParameters(Parameters):
+    """CzFluxTime runcard inputs."""
+
+    amplitude_factor_min: float
+    """Amplitude minimum."""
+    amplitude_factor_max: float
+    """Amplitude maximum."""
+    amplitude_factor_step: float
+    """Amplitude step."""
+    duration_min: float
+    """Duration minimum."""
+    duration_max: float
+    """Duration maximum."""
+    duration_step: float
+    """Duration step."""
+    pairs: Union[list[Tuple[int], Tuple[int]]]
+    """Pair(s) of qubit to probe."""
```

Comment:
Following my suggestion this should be:

```suggestion
    qubits: List[List[QubitId,QubitId]]
    """Pair(s) of qubit to probe."""
```
where you can import `QubitId` from qibolab.qubits